src/ImageClassifier/index.test.js

```diff
@@ -1,50 +1,44 @@
 /* eslint new-cap: 0 */
 
-import * as ImageClassifier from './index';
-
-const DEFAULTS = {
-  learningRate: 0.0001,
-  hiddenUnits: 100,
-  epochs: 20,
-  numClasses: 2,
-  batchSize: 0.4,
+import imageClassifier from './index';
```

Comment:
cc @cvalenzuela one really useful thing to unit tests is memory usage.

One pattern we do is something like this:

```
const numTensors = tf.memory().numTensors;
apiCall();
expect(tf.memory().numTensors).toEqual(numTensors);
```

This makes sure you don't introduce leaks.

Reply:
great!, I didn't knew that.